public/app/features/integrated-alerting/components/AlertRuleTemplate/AlertRuleTemplatesTable/AlertRuleTemplatesTable.styles.ts

```diff
@@ -0,0 +1,85 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { selectThemeVariant, stylesFactory } from '@grafana/ui';
+
+export const getStyles = stylesFactory((theme: GrafanaTheme) => {
+  const backgroundColor = selectThemeVariant({ light: 'rgb(247, 247, 249)', dark: '#161719' }, theme.type);
```

Comment:
```suggestion
  const backgroundColor = selectThemeVariant({ light: '#f7f7f9', dark: '#161719' }, theme.type);
```

Reply:
Or better yet, use the colors from the pallette, as Tiago mentioned.

Reply:
done